float32 goal_position
float32 goal_velocity
float32 goal_effort
int32 control_mode
int32 inturrpt
